source/parser/parser.cpp

```diff
@@ -2911,7 +2911,7 @@ ObjectPtr Parser::Parse_Julia_Fractal ()
 
 ObjectPtr Parser::Parse_Lathe()
 {
-    int i;
+    int i, AlreadyWarned;
```

Comment:
Can you make this a bool, and adjust the test and modification of this variable accordingly? I don't think we need to count how many times we have issued a warning.